client/src/components/postcall/OneQuestionPollInput.test.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { OneQuestionPollInput, OneQuestionPollInputProps } from './OneQuestionPollInput';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { IconButton, Rating, setIconOptions, TextField } from '@fluentui/react';
+
+configure({ adapter: new Adapter() });
+// Disable icon warnings for tests as we don't register the icons for unit tests which causes warnings.
+// See: https://github.com/microsoft/fluentui/wiki/Using-icons#test-scenarios
+setIconOptions({
+  disableWarnings: true
+});
+describe('OneQuestionPollInput', () => {
+  it('should trigger setPollResponse when like selected', async () => {
+    const mockPollType = 'likeOrDislike';
+    const mockTextInputPlaceholder = 'mock';
+    const mockSetPollResponse = jest.fn();
+
+    const onQuestionPollInput = await mount<OneQuestionPollInputProps>(
+      <OneQuestionPollInput
+        pollType={mockPollType}
+        textInputPlaceholder={mockTextInputPlaceholder}
+        setPollResponse={mockSetPollResponse}
+      />
+    );
+    const iconButton = onQuestionPollInput.find(IconButton);
+    expect(iconButton.length).toBe(2);
+    iconButton.first().simulate('click');
+    expect(mockSetPollResponse).toBeCalledWith(true);
+  });
+
+  it('should trigger setPollResponse when rating changed', async () => {
+    const mockPollType = 'rating';
+    const mockTextInputPlaceholder = 'mock';
+    const mockSetPollResponse = jest.fn();
+
+    const onQuestionPollInput = await mount<OneQuestionPollInputProps>(
+      <OneQuestionPollInput
+        pollType={mockPollType}
+        textInputPlaceholder={mockTextInputPlaceholder}
+        setPollResponse={mockSetPollResponse}
+      />
+    );
+    const rating = onQuestionPollInput.find(Rating);
+    expect(rating.length).toBe(1);
+    rating.simulate('change');
```

Comment:
Is it possible to provide an actual value here so we can assert that `mockSetPollResponse` was called with that value?

Reply:
Nope , tried various combinations of click and rating and value and that doesnt seem to do the trick.

Reply:
How come you were able to do it here? https://github.com/Azure-Samples/communication-services-virtual-visits-js/pull/98/files#diff-c9dc5310ac40d64d672e151215e4fb7625f913dbc00ad6a070bccff186a3cdbbR68

Reply:
Rating in this instance is a bunch of 5 different rating buttons.. it will involve finding a specific one and triggering the right event. Triggering change and click for the GROUP level element as well as the first rating button did not work .

As of now test coverage for this file is at 91.89 % . I think we can leave this